modules/govuk/templates/node/s_asset_base/process-uploaded-attachments.sh.erb

```diff
@@ -5,25 +5,31 @@ set -e
 INCOMING_DIR=$1
 CLEAN_DIR=$2
 INFECTED_DIR=$3
+TMP_DIR=$4
 
 set -u
 
 usage() {
-  echo "USAGE: $0 <incoming_directory> <clean_directory> <infected_directory>"
+  echo "USAGE: $0 <incoming_directory> <clean_directory> <infected_directory> <tmp_dir>"
   echo
   echo "Processes attachments in <incoming_directory> by:"
   echo "  - Running a virus scan on each attachment"
   echo "  - If the file is clean, moving it to a directory of clean files"
-  echo "  - Copying it to one or more other machines"
+  echo "  - Create a file list of outstanding files in <tmp_dir> to sync to asset slaves"
   echo
   exit 1
 }
 
-if [ $# -lt 3 ]; then
+if [ $# -lt 4 ]; then
  usage
 fi
 
-ASSET_SLAVE_NODES=$(/usr/local/bin/govuk_node_list -c asset_slave)
+# Creates a unique filename like "filelist-mnt-uploads-whitehall-clean-1479475223"
+FILE_LIST="${TMP_DIR}/filelist$(echo ${CLEAN_DIR} | sed 's/\//-/g')-$(date +%s)"
+
+if [[ ! -d ${TMP_DIR} ]]; then
```

Comment:
Does this need to be a conditional if we're going to create it everytime it doesn't exist?